actions/index.js

```diff
@@ -0,0 +1,7 @@
+export function AddItemAction(modalText, response) {
```

Comment:
Action creators must be at `camelCase`. Also, you can be more specific here, with something like: `addRepository`.